docs/content/en/docs/pipeline-stages/init.md

```diff
@@ -164,7 +164,7 @@ For multiple artifacts, use `--artifact` multiple times.
 ```bash
 microservices$skaffold init \
```

Comment:
Can we drop the `microservices$ ` here?  We usually only show a prompt to distinguish with output.

Reply:
done!